scripts/cli.sh

```diff
@@ -0,0 +1,112 @@
+#! /bin/bash
```

Comment:
We should change this to:
```suggestion
#! /usr/bin/env bash
```
This can be helpful for certain environments and will be consistent with other scripts in the codebase.